scripts/projects.js

```diff
@@ -20,24 +20,32 @@ function Project(itm){
   this.authors = itm.authors;
 }
 
-Project.prototype.toHtml = function() {
+Project.prototype.toHtml = function(el, inx, arry) {
+  //delcare local variables
   var finAuthors = '';
+  var finProj = '';
   var authorTemplate = $('#authTemp').html();
   var projectTemplate = $('#projTemp').html();
   var dataSource = this;
-  console.log(dataSource);
   var compAuth = Handlebars.compile(authorTemplate);
   var compProj = Handlebars.compile(projectTemplate);
+  //Place the heading before the first item only
+  if(inx==0){
+    finProj = "<h1>Projects I've made</h1><hr />"
+  }
   //Compile for each author.
   dataSource.authors.forEach(function(ele, index, arr){
     finAuthors += compAuth(ele);
     if(index!=arr.length-1){
-      finAuthors += " and ";
+      finAuthors += " and "; //Place " and " after each author that isn't the last one. We don't need to do anything for the last author.
     };
   });
   this.authorPlaceholder = finAuthors;
-  var finProj = compProj(dataSource)+"<hr/>";
-  console.log(finProj);
+  if(inx!=arry.length-1){
+    finProj += compProj(dataSource)+"<hr/>";//Place horizontal rows after each project except last
```

Comment:
@Ansem717 <hr /> and <br /> tags should be avoided as the stylistic (and spacing) aspect of content should managed in CSS